src/frontend_s3.py

```diff
@@ -29,8 +30,23 @@ def create_frontend_bucket(self):
             ),
             force_destroy=self.frontend_config.require_bool("s3_force_destroy"),
         )
+        self.s3_bucket = bucket
         return bucket
 
+    def render_config_to_s3_bucket(self, backend_uri):
+        config_content = backend_uri.apply(
+            lambda uri: f"""
+            {{
+                \"backendApiUrl\": \"https://{uri}\"
+            }}
+            """
```

Comment:
You can easily generate json string with a simple python dict:
```python
import json
config_content = json.dumps({"backendApiUrl":f"http://{backend_uri}"})
```

Reply:
Good point. Fixed